apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -668,10 +674,21 @@ class TopInstructions extends Component {
           >
             <div ref="instructions">
               {this.props.hasContainedLevels && (
```

Comment:
worth considering briefly whether this should also be `showContainedLevelAnswer` ?